backend/app/usecase/repository/shortlink_fake.go

```diff
@@ -12,6 +12,14 @@ var _ ShortLink = (*ShortLinkFake)(nil)
 // ShortLinkFake accesses ShortLink information in short_link table through SQL.
 type ShortLinkFake struct {
 	shortLinks map[string]entity.ShortLink
+	// TODO(issue#958) use eventbus for propagating short link change to user short link repo
```

Comment:
```suggestion
	// TODO(issue#958) use eventbus for propagating short link change to all related repos
```

Reply:
Yes that makes sense, as user short link is not the only thing connected to short link repo, like we discussed yesterday. Thanks for this suggestion!